plugin/src/main/java/org/opensearch/ml/action/prediction/TransportPredictionTaskAction.java

```diff
@@ -180,6 +186,67 @@ public void onFailure(Exception e) {
         }
     }
 
+    private void deployAndPredictRemoteModel(
+        String modelId,
+        ActionListener<MLTaskResponse> wrappedListener,
+        MLPredictionTaskRequest mlPredictionTaskRequest
+    ) {
+        String[] workerNodes = mlModelManager.getWorkerNodes(modelId, FunctionName.REMOTE, true);
```

Comment:
Should we use `true` or false for `getWorkerNodes` here?

Reply:
should be true here. Eligible nodes for remote models are ml nodes and data nodes. Also that's how it's used elsewhere in the "predict".